Cython/Compiler/Options.py

```diff
@@ -235,6 +235,7 @@ def copy_inherited_directives(outer_directives, **new_directives):
     'warn.unused_arg': False,
     'warn.unused_result': False,
     'warn.multiple_declarators': True,
+    'warn.deprecated_if': True,
```

Comment:
Assuming that there might be more deprecation warnings to hide at some point, I propose this naming scheme:
```suggestion
    'warn.deprecated.IF': True,
```
And then we should probably also add `warn.deprecated.DEF`.

Reply:
Looks like the `DEF` is currently commented out, so probably no need to add it here?

Reply:
I'd go the other way: if we can add an option to enable/disable that warning, then we should comment it back in and disable the warning using the new option instead.

Reply:
Done!